bundle/edu.gemini.itc.web/src/main/scala/edu/gemini/itc/web/ITCRequest.scala

```diff
@@ -198,22 +198,13 @@ object ITCRequest {
   // GHOST
   def ghostParameters(r: ITCRequest): GhostParameters = {
 
-    def extractGain(r:GhostType.ReadMode) : GhostType.AmpGain = r match {
-      case GhostType.ReadMode.SLOW_LOW => GhostType.AmpGain.LOW
-      case GhostType.ReadMode.MEDIUM_LOW => GhostType.AmpGain.LOW
-      case GhostType.ReadMode.FAST_LOW => GhostType.AmpGain.LOW
-    }
-
-    val spatBinning              = r.enumParameter(classOf[GhostType.Binning],"spatialBinning");
-    Log.info("spatBinning: " + spatBinning);
-    val centralWl                = r.centralWavelengthInNanometers();
-    val specBinning              = r.enumParameter(classOf[GhostType.Binning],"spectralBinning");
-    val readMode                 = r.enumParameter(classOf[GhostType.ReadMode]);
-    val ampGain                  = extractGain(readMode);
-    val resolution               = r.enumParameter(classOf[GhostType.Resolution],"instResolution");
-    val nSkyMicrolens            =  ghostGetNumSky(r); //r.intParameter("nSkyMicrolens");
+    val binning       = r.enumParameter(classOf[GhostBinning],"binning")
+    val centralWl     = r.centralWavelengthInNanometers()
+    val readMode      = r.enumParameter(classOf[GhostReadNoiseGain], "ReadMode")
+    val resolution    = r.enumParameter(classOf[ResolutionMode],"instResolution")
+    val nSkyMicrolens = ghostGetNumSky(r)
```

Comment:
We don't have this parameter on the OT model. We probably need to add it before integrating the OT with ITC